ArticleTemplates/assets/js/boot.js

```diff
@@ -96,19 +96,28 @@ const init = opts => {
             return 'default';
         };
 
+        const getHideAdsTest = (tests) => {
+            if (!tests) return 0;
+            const parsedTests = JSON.parse(tests);
```

Comment:
Occurs to me that `JSON.parse` can throw an exception, is it worth wrapping in `try...catch`?